contracts/Market.sol

```diff
@@ -1101,10 +1108,13 @@ contract Market is
    * @notice Set the price multiple, which is the number of base tokens required to purchase one NRT.
    * @dev This value is scaled by 100 to allow for decimal precision. For example, a value of 100 means
    * that 1 base token is required to purchase 1 NRT, while a value of 1995 means that 19.95 base tokens
-   * purchase 1 NRT.
+   * purchase 1 NRT. The minimum value for the price multiple is 100, to avoid loss of decimal precision.
    * @param priceMultiple The new price multiple.
    */
   function _setPriceMultiple(uint256 priceMultiple) internal {
+    if (priceMultiple < 100) {
+      revert InvalidPriceMultiple();
+    }
```

Comment:
There actually is a test that covers this line of code ... I think there's an issue with codecov picking this up correctly perhaps related to our forge testing patterns with helper contracts?

Reply:
Perhaps a slightly different error, though I'm surprised it wasnt counted as covered in this case

Reply:
possible it's [this](https://github.com/foundry-rs/foundry/issues/4316) bug?

Reply:
interesting... that seems possible since the test is in a test contract that inherits `NonUpgradeableMarket` which inherits `Market` -- this is different from the pattern in most of our tests that inherit `UpgradeableMarket` which does not inherit the target contracts but rather has instances of them I believe.... (and for which coverage seems to be working more as expected)